src/platform/forms/tests/fixtures/mocks/mockUser.js

```diff
@@ -0,0 +1,61 @@
+import moment from 'moment';
+import { VA_FORM_IDS } from 'platform/forms/constants';
+/* eslint-disable camelcase */
+
+const mockUser = {
+  data: {
+    attributes: {
+      profile: {
+        sign_in: {
+          service_name: 'idme',
+        },
+        email: 'fake@fake.com',
+        loa: { current: 3 },
+        first_name: 'Jane',
+        middle_name: '',
+        last_name: 'Doe',
+        gender: 'F',
+        birth_date: '1985-01-01',
+        verified: true,
+      },
+      veteran_status: {
+        is_veteran: true,
+        status: 'OK',
+        served_in_military: true,
+      },
+      in_progress_forms: [
+        {
+          form: VA_FORM_IDS.FORM_10_10EZ,
+          last_updated: 1501608808,
+          metadata: {
+            last_updated: 1506792808,
```

Comment:
Just confirming it's ok if these values are hardcoded and not derived via `moment` like the `expires_at` 🙂

Reply:
At present, our objective is more of a "lift and shift" to get us away from Nightwatch and having to run two testing frameworks side by side at deployment. This was how it was being done in Nightwatch so I've copied it over here.  Happy to explore reevaluating in a future initiative